src/doc/DocAnnotator.js

```diff
@@ -779,8 +790,12 @@ class DocAnnotator extends Annotator {
             return false;
         }
 
-        // $FlowFixMe
-        if (this.createHighlightDialog.isVisible) {
+        // Does nothing if the use is creating a highlight or has double clicked text
+        if (
+            (this.createHighlightDialog && this.createHighlightDialog.isVisible) ||
+            // $FlowFixMe
+            event.detail === 2
```

Comment:
Not exactly related to this change but can you update the jsdoc for the return value?